sdk/storage/azure-storage-files-shares/test/share_client_test.cpp

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 #include "share_client_test.hpp"
+#include "azure/storage/common/crypt.hpp"
```

Comment:
reformat

Reply:
Don't know why my Clang formatter does not catch this one.